airbyte-integrations/connectors/source-mailchimp/source_mailchimp/client.py

```diff
@@ -62,25 +70,72 @@ def health_check(self):
 
     def get_streams(self):
         streams = []
-        for entity in self._entities:
-            json_schema = requests.get(self.API_MAILCHIMP_URL.format(entity)).json()
-            streams.append(AirbyteStream(name=entity, json_schema=json_schema))
+        for entity in self._ENTITIES:
+            raw_schema = json.loads(pkgutil.get_data(self.__class__.__module__.split(".")[0], f"schemas/{entity}.json"))
+            streams.append(AirbyteStream.parse_obj(raw_schema))
         return streams
 
-    def lists(self):
-        limit = self._client.lists.all(count=1)["total_items"]
+    def lists(self, state: DefaultDict[str, any] = None) -> Generator[AirbyteMessage, None, None]:
```

Comment:
Not DRY'ing until we add support for child/nested entities because those do not have the same structure as these entities.